src/Compilers/CSharp/Test/Semantic/Semantics/PatternMatchingTests3.cs

```diff
@@ -5501,5 +5501,107 @@ static void Main()
                 );
             var compVerifier = CompileAndVerify(compilation, expectedOutput: expectedOutput);
         }
+
+        [Fact, WorkItem(44518, "https://github.com/dotnet/roslyn/issues/44518")]
+        public void Crash_01()
+        {
+            var source =
+@"class C
+{
+    void M()
+    {
+        _ = is < true;
+    }
+}
+";
+            var compilation = CreateCompilation(source, parseOptions: TestOptions.RegularWithPatternCombinators);
+            compilation.VerifyDiagnostics(
+                // (5,13): error CS1525: Invalid expression term 'is'
+                //         _ = is < true;
+                Diagnostic(ErrorCode.ERR_InvalidExprTerm, "is").WithArguments("is").WithLocation(5, 13)
```

Comment:
>ERR_InvalidExprTerm [](start = 37, length = 19)

These errors look like parse errors. Do we hit the binding code with this test? Same comment for tests below.

Reply:
This test did crash without the fix. See https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQBMQGoA+ABATAAgGEBYAKAG8z9r9sAWfAWQAoBKKmy0mn/AfXwBefAEso+ADz4YCAK5wA3B2oBfMiqA